Scrips.BaseDbContext/Scrips.BaseDbContext/ApplicationDbContext.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Dapper;
+using IdentityModel;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Scrips.BaseDbContext.Dtos;
+using Scrips.BaseDbContext.Entities;
+using Serilog;
+using System.Data;
+
+namespace Scrips.BaseDbContext
+{
+    public class ApplicationDbContext : DbContext
```

Comment:
Better to have a more descriptive name like BaseDbContext or AuditableBaseDbContext

Reply:
Change to AuditableBaseDbContext